src/extension/wps/web-wps/src/main/java/org/geoserver/wps/web/ProcessStatusProvider.java

```diff
@@ -20,14 +40,22 @@
  */
 @SuppressWarnings("serial")
 public class ProcessStatusProvider extends GeoServerDataProvider<ExecutionStatus> {
+    static private Logger LOGGER = Logging.getLogger(ProcessStatusProvider.class);
+    static private final FilterFactory2 FF = CommonFactoryFinder.getFilterFactory2();
     static final Property<ExecutionStatus> TYPE = new AbstractProperty<ExecutionStatus>("type") {
-
         @Override
         public Object getPropertyValue(ExecutionStatus item) {
             // we might want to have a "C" state for the chained processes
             return item.isAsynchronous() ? "A" : "S";
         }
 
+        @Override
+        public boolean isSearchable() {
+            // we really it isn't sortable
```

Comment:
The comment does not make much sense written like this. I guess you wanted to say it's not searchable?